torch/csrc/api/src/serialize/input-archive.cpp

```diff
@@ -70,12 +56,11 @@ void InputArchive::read(
 }
 
 bool InputArchive::try_read(const std::string& key, InputArchive& archive) {
-  if (auto named_module = module_.find_module(key)) {
-    archive.module_ = std::move(*named_module);
-    return true;
-  } else {
+  if (!module_.hasattr(key)) {
     return false;
   }
+  archive.module_ = module_.attr(key).toModule();
```

Comment:
Wouldn't we crash here if the attribute is there, but it's not a module? In the original code we returned `false` in this case.

Reply:
Yeah, I can make this more specific.